Sources/LocalizedStringKit/LocalizedStringKit.m

```diff
@@ -101,30 +105,42 @@ + (NSString *)keyWithValue:(NSString *_Nonnull)value keyExtension:(NSString *)ke
   return key;
 }
 
-+ (NSBundle *)getLocalizedStringKitBundle:(NSString *_Nullable)tableName
++ (NSBundle *)getLocalizedStringKitBundle:(NSString *_Nullable)bundleName
 {
+  // Search Paths
   NSURL *searchPath = [[NSBundle mainBundle] bundleURL];
 
-  if (tableName == nil) {
-    // Defaults to primary bundle if tableName not specified
-    tableName = @"LocalizedStringKit.bundle";
+  // Determine target bundleName
+  if (bundleName == nil) {
+    // Defaults to primary bundle if bundleName not specified
+    bundleName = primaryBundleName;
   }
   else
   {
     // Append suffix
-    tableName = [tableName stringByAppendingFormat:@".bundle"];
+    bundleName = [bundleName stringByAppendingFormat:@".bundle"];
   }
 
   while(YES)
   {
-    NSURL *bundleURL = [searchPath URLByAppendingPathComponent:tableName];
+    // Primary searchPath check
+    NSURL *bundleURL = [searchPath URLByAppendingPathComponent:bundleName];
     NSBundle *bundle = [NSBundle bundleWithURL:bundleURL];
 
     if (bundle)
     {
       return bundle;
     }
 
+    // Alternate path check, if url specified
+    if (alternateBundleSearchPath != nil) {
```

Comment:
This is going to be checked every time we run the loop. 

It's in a loop because for extensions, `Bundle.main` returns the extension bundle, which is inside the main app bundle. So if we don't find the localized string .bundle in there, we go up a level and check again until we do find it.

Reply:
Woops, I'll move it out of the loop.